test/rtos_drivers/hil_add/src/individual_tests/uart/uart_test.h

```diff
@@ -0,0 +1,34 @@
+// Copyright 2021 XMOS LIMITED.
+// This Software is subject to the terms of the XMOS Public Licence: Version 1.
+
+#ifndef UART_TEST_H_
+#define UART_TEST_H_
+
+#include "rtos_test/rtos_test_utils.h"
+
+#define uart_printf( FMT, ... )       module_printf("UART", FMT, ##__VA_ARGS__)
+
+#define UART_MAX_TESTS   1
+
+// #define SPI_MAIN_TEST_ATTR          __attribute__((fptrgroup("rtos_test_spi_main_test_fptr_grp")))
+// #define SPI_SLAVE_XFER_DONE_ATTR    __attribute__((fptrgroup("rtos_test_spi_slave_xfer_done_fptr_grp")))
```

Comment:
Why have these symbols been left in but commented out?

Reply:
Just a mistake. I adapted much of the work from existing components.